jwst/assign_wcs/tests/test_wfss.py

```diff
@@ -0,0 +1,392 @@
+import numpy as np
+from numpy.testing import assert_allclose
+
+from astropy.modeling.models import (Polynomial1D, Polynomial2D, Shift,
+                                     Const1D, Mapping)
+from gwcs import wcs
+from gwcs.wcstools import grid_from_bounding_box
+
+from ...datamodels import SlitModel
+from ...transforms import models as transforms
+from ...extract_2d.grisms import compute_wavelength_array
+
+import pytest
+
+
+def create_slit(model, x0, y0, order):
+    """ Create a SlitModel representing a grism slit."""
+    ymin = 0
+    xmin = 0
+    # ymax = 58
+    # xmax = 1323
+    model = Mapping((0, 1, 0, 0, 0)) | (Shift(xmin) & Shift(ymin) &
+                                        Const1D(x0) & Const1D(y0) & Const1D(order) )| model
+    wcsobj = wcs.WCS([('det', model), ('world', None)])
+    # refactored call
+    # wcsobj.bounding_box = ((3, 7), (2, 5), (x0-1, x0+1), (y0-1, y0+1), (1, 3))
+
+    #wcsobj.bounding_box = ((xmin, xmax), (ymin, ymax))
+    wcsobj.bounding_box = ((20, 25), (800, 805))
```

Comment:
These values don't match your values for xmin, xmax, ymin, ymax above. Is that OK?

Reply:
I didn't want to compare big arrays just a few numbers.